get-deps

```diff
@@ -223,22 +223,10 @@ gentoo_deps() {
   portageq envvar USE | xargs -n 1 | grep '^X$' \
   || (echo 'X is not found in USE flags' && exit 1)
   EMERGE="$SUDO emerge"
-  $EMERGE --select \
-    'cmake' \
-    'fontconfig' \
-    'openssl' \
-    'git' \
-    'libX11' \
-    'libxkbcommon' \
-    'pkgconf' \
-    'python' \
-    'perl' \
-    'wayland' \
-    'mesa' \
-    'xcb-util' \
-    'xcb-util-image' \
-    'xcb-util-keysyms' \
-    'xcb-util-wm'
+  for i in 'cmake' 'fontconfig' 'openssl' 'dev-vcs/git' 'libX11' 'libxkbcommon' 'pkgconf' 'python' 'perl' 'wayland' 'mesa' 'xcb-util' 'xcb-util-image' 'xcb-util-keysyms' 'xcb-util-wm'
+  do
+	  equery l "$i" > /dev/null || emerge --select $i
```

Comment:
please use `$EMERGE` rather than `emerge` so that `sudo` is used when appropriate:

```suggestion
	  equery l "$i" > /dev/null || $EMERGE --select $i
```